source/opt/ir_context.cpp

```diff
@@ -339,6 +339,12 @@ void IRContext::AddCombinatorsForCapability(uint32_t capability) {
         SpvOpImageDrefGather,
         SpvOpImageRead,
         SpvOpImage,
+        SpvOpImageQueryFormat,
+        SpvOpImageQueryOrder,
+        SpvOpImageQuerySizeLod,
+        SpvOpImageQuerySize,
+        SpvOpImageQueryLevels,
+        SpvOpImageQuerySamples,
```

Comment:
There was a TODO(dneto) about ImageQuery at the end.  I think that's safe to delete now.

Reply:
Done.